tests/Shared/WorkloadTesting/BuildEnvironment.cs

```diff
@@ -34,36 +33,42 @@ public class BuildEnvironment
     public static BuildEnvironment ForNet80 => s_instance_80.Value;
     public static BuildEnvironment ForDefaultFramework => ForNet80;
 
-    public BuildEnvironment(bool useSystemDotNet = false, TestTargetFramework targetFramework = DefaultTargetFramework)
+    public BuildEnvironment(bool useSystemDotNet = true, TestTargetFramework targetFramework = DefaultTargetFramework, bool installTemplates = true)
     {
+        if (!useSystemDotNet)
+        {
+            Console.WriteLine($"** IGNORING unsupported config with useSystemDotNet={useSystemDotNet}");
+            useSystemDotNet = true;
+        }
+
         HasWorkloadFromArtifacts = !useSystemDotNet;
         TargetFramework = targetFramework;
         RepoRoot = TestUtils.FindRepoRoot();
 
-        string sdkForWorkloadPath;
+        // string sdkForWorkloadPath;
         if (RepoRoot is not null)
         {
             // Local run
-            if (!useSystemDotNet)
-            {
-                var sdkDirName = string.IsNullOrEmpty(EnvironmentVariables.SdkDirName) ? "dotnet-latest" : EnvironmentVariables.SdkDirName;
-                var sdkFromArtifactsPath = Path.Combine(RepoRoot!.FullName, "artifacts", "bin", sdkDirName);
-                if (Directory.Exists(sdkFromArtifactsPath))
-                {
-                    sdkForWorkloadPath = Path.GetFullPath(sdkFromArtifactsPath);
-                }
-                else
-                {
-                    string buildCmd = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? ".\\build.cmd" : "./build.sh";
-                    string workloadsProjString = Path.Combine("tests", "workloads.proj");
-                    throw new XunitException(
-                        $"Could not find a sdk with the workload installed at {sdkFromArtifactsPath} computed from {nameof(RepoRoot)}={RepoRoot}." +
-                        $" Build all the packages with '{buildCmd} -pack'." +
-                        $" Then install the sdk+workload with 'dotnet build {workloadsProjString}'." +
-                        " See https://github.com/dotnet/aspire/tree/main/tests/Aspire.Workload.Tests#readme for more details.");
-                }
-            }
-            else
+            // if (!useSystemDotNet)
```

Comment:
@radical should this be removed? IF not I can log an issue to make sure we are tracking this follow-up work.

Reply:
I'm already working on the follow up in the templates9 PR, which includes changes from this branch.

Reply:
Ok great, then I'll leave as-is.